### PR TITLE
Add Run Tasks Entitlement to API Docs

### PR DESCRIPTION
**Pull request by glennsarti**
Previously Run Tasks was a feature flag but it is now a proper organization
entitlement.  This commit updates the various feature set and organization
entitlement examples with the new `run-tasks` entitlement, and updates
the feature set list with what this new entitlement affects.

[RFC](https://docs.google.com/document/d/1fsZI2IC8GtXtf_rcYV8kqD2xRifZfYdXAVSEwFoetMc/edit#)

### Screenshots

example `cloud-docs/api-docs#feature-entitlement`
image

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] Redirects have been added for moved, renamed, or deleted pages.
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate.
- [x] New pages have metadata (page name and description) at the top.
- [x] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Comment by glennsarti** [comment]
Added @jarrodn-au  for technical review and @laurapacilio for grammar/content review.

**Comment by glennsarti** [comment]
Asana Ticket - https://app.asana.com/0/1202068080217073/1202084424923848/f

File: content/cloud-docs/api-docs/index.mdx
Comment (glennsarti) [inline content/cloud-docs/api-docs/index.mdx:89]: @laurapacilio   This is probably the only "controversial" bit of the PR.  I wasn't quite sure how to distinguish between `run tasks` the integration/feature versus  `run tasks` the API endpoint.

That was the best I could come up with. I also tried `Allows an organization to use Run Task Integrations ....` but I remember in the last PR, features tend not to have uppercase names.  And I can see cost estimates and vcs are both lower case.

**Review by jarrodn-au — APPROVED** [review]

File: content/cloud-docs/api-docs/index.mdx
Comment (laurapacilio) [reply content/cloud-docs/api-docs/index.mdx:89]: thank you for flagging this and for caring about capitalization (that makes me happy!). I support the decision to stick with lower case because first, as you mentioned, we do not capitalize feature names. And second, we don't capitalize the things anywhere else here - consistency is good. My only question is actually - I see you're linking to the integrations page, but would it be more helpful  to link here: https://www.terraform.io/cloud-docs/workspaces/settings/run-tasks

The reason why is that the integrations page is really (at least to me) about how to build a run task integration for TFC. But I think we may want the more end user context? Lmk if you completely disagree though!

**Review by laurapacilio — APPROVED** [review]
Approving so I don't block, but refer to my minor comment! Thank you!!

File: content/cloud-docs/api-docs/index.mdx
Comment (glennsarti) [reply content/cloud-docs/api-docs/index.mdx]: Agreed. That's a better link!

Comment (glennsarti) [reply content/cloud-docs/api-docs/index.mdx]: Updated

**Comment by glennsarti** [comment]
Thanks @laurapacilio .  I won't merge quite yet.... This is one of those rare times when the docs are ready before the feature 😲  I'll merge once the feature goes live.

**Review by cam-stitt — APPROVED** [review]
This should be good to merge now.

**Comment by glennsarti** [comment]
Merging.

**Pull request merged**
